app/views/volunteers/show.html.slim

```diff
@@ -7,83 +7,103 @@
     .table-responsive
       table.table.table-no-border-top
         tbody
-          tr
-            td= t_attr(:date_of_birth)
-            td= l(@volunteer.date_of_birth) if @volunteer.date_of_birth
-          tr
-            td= t_attr(:gender)
-            td= t("gender.#{@volunteer.gender}") unless @volunteer.gender.blank?
-          tr
-            td= t_attr(:avatar)
-            td= image_tag @volunteer.avatar.url(:thumb) if @volunteer.avatar.present?
-          tr
-            td= t_attr(:nationality)
-            td= country(@volunteer.nationality)
-          tr
-            td= t_attr(:additional_nationality)
-            td= country(@volunteer.additional_nationality)
-          tr
-            td= t_attr(:street)
-            td= @volunteer.street
-          tr
-            td= t_attr(:zip)
-            td= @volunteer.zip
-          tr
-            td= t_attr(:city)
-            td= @volunteer.city
-          tr
-            td= t_attr(:email)
-            td= @volunteer.email
-          tr
-            td= t_attr(:phone)
-            td= @volunteer.phone
-
-          = render 'language_skills/show', speaker: @volunteer
-          = render 'relatives/show', relative: @volunteer
-
-          tr
-            td= t_attr(:profession)
-            td= @volunteer.profession
-          tr
-            td= t_attr(:education)
-            td= @volunteer.education
-          tr
-            td= t_attr(:motivation)
-            td= @volunteer.motivation
-          tr
-            td= t_attr(:experience)
-            td= Volunteer.human_boolean(@volunteer.experience)
-          tr
-            td= t_attr(:expectations)
-            td= @volunteer.expectations
-          tr
-            td= t_attr(:strengths)
-            td= @volunteer.strengths
-          tr
-            td= t_attr(:skills)
-            td= @volunteer.skills
-          tr
-            td= t_attr(:interests)
-            td= @volunteer.interests
-          tr
-            td= t_attr(:state)
-            td= @volunteer.state
-          tr
-            td= t_attr(:duration)
-            td= t("duration.#{@volunteer.duration}") unless @volunteer.duration.blank?
-          tr
-            td= t_attr(:region)
-            td= t("region.#{@volunteer.region}") unless @volunteer.region.blank?
+          - if @volunteer.date_of_birth.present?
```

Comment:
This is part of a UX question, i prefer to see the empty fields, because when i know how the view looks and then don't see the keys, i get the feeling that something is wrong.

Second is to check every field with `.present?` is a lot of repetition, this could be done with a helper method